logs_api/lib/opentelemetry/logs/logger.rb

```diff
@@ -12,36 +12,45 @@ class Logger
 
       # Emit a {LogRecord} to the processing pipeline.
       #
-      # @param timestamp [optional Float, Time] Time in nanoseconds since Unix
+      # @param [optional Float, Time] timestamp Time in nanoseconds since Unix
       #   epoch when the event occurred measured by the origin clock, i.e. the
       #   time at the source.
-      # @param observed_timestamp [optional Float, Time] Time in nanoseconds
+      # @param [optional Float, Time] observed_timestamp Time in nanoseconds
       #   since Unix epoch when the event was observed by the collection system.
       #   Intended default: Process.clock_gettime(Process::CLOCK_REALTIME)
-      # @param context [optional Context] The Context to associate with the
-      #   LogRecord. Intended default: OpenTelemetry::Context.current
-      # @param severity_number [optional Integer] Numerical value of the
+      # @param [optional Integer] severity_number Numerical value of the
       #   severity. Smaller numerical values correspond to less severe events
       #   (such as debug events), larger numerical values correspond to more
       #   severe events (such as errors and critical events).
-      # @param severity_text [optional String] Original string representation of
+      # @param [optional String] severity_text Original string representation of
       #   the severity as it is known at the source. Also known as log level.
-      # @param body [optional String, Numeric, Boolean, Array<String, Numeric,
+      # @param [optional String, Numeric, Boolean, Array<String, Numeric,
       #   Boolean>, Hash{String => String, Numeric, Boolean, Array<String,
-      #   Numeric, Boolean>}] A value containing the body of the log record.
-      # @param attributes [optional Hash{String => String, Numeric, Boolean,
-      #   Array<String, Numeric, Boolean>}] Additional information about the
-      #   event.
+      #   Numeric, Boolean>}] body A value containing the body of the log record.
+      # @param [optional String] trace_id The trace ID associated with the
+      #   current context.
+      # @param [optional String] span_id The span ID associated with the
+      #   current context.
+      # @param [optional TraceFlags] trace_flags The trace flags associated
+      #   with the current context.
+      # @param [optional Hash{String => String, Numeric, Boolean,
+      #   Array<String, Numeric, Boolean>}] attributes Additional information
+      #   about the event.
+      # @param [optional Context] context The Context to associate with the
+      #   LogRecord. Intended default: OpenTelemetry::Context.current
       #
       # @api public
-      def emit(
+      def on_emit(
```

Comment:
When we renamed the method in https://github.com/open-telemetry/opentelemetry-ruby/pull/1517, I forgot to go back and change the references in the API.